cmd/rook/ceph/operator.go

```diff
@@ -17,8 +17,11 @@ limitations under the License.
 package ceph
 
 import (
+	b64 "encoding/base64"
```

Comment:
why is this renaming required? Original package name `base64` is good enough.

Reply:
Fixed